test/tests.hpp

```diff
@@ -105,6 +105,12 @@ inline std::ostream& outputIndependentBits(std::ostream& os, Qrack::QInterfacePt
 class QInterfaceTestFixture {
 protected:
     Qrack::QInterfacePtr qftReg;
+    /*
+     * Default engine type to run the tests with. Global because catch doesn't
+     * support parameterization.
+     */
+    enum Qrack::QInterfaceEngine engineType;
```

Comment:
These don’t seem to be used anywhere; additionally, the comment is wrong.

Reply:
Removed.